kolla/node_custom_config/grafana.ini

```diff
@@ -0,0 +1,21 @@
+[server]
+root_url = "https://{{kolla_external_fqdn}}:3000"
+
+[auth]
+disable_login_form = "true"
+oauth_auto_login = "true"
+
+[auth.generic_oauth]
+name = Chameleon
+icon = signin
+enabled = true
+allow_sign_up = true
+client_id = "{{ keystone_idp_client_id }}"
+scopes = openid profile email
+auth_url = "https://auth.chameleoncloud.org/auth/realms/chameleon/protocol/openid-connect/auth"
+token_url = "https://auth.chameleoncloud.org/auth/realms/chameleon/protocol/openid-connect/token"
+api_url = "https://auth.chameleoncloud.org/auth/realms/chameleon/protocol/openid-connect/userinfo"
+
+role_attribute_path = contains(projects[*].id, 'Chameleon') && 'Admin' || contains(projects[*].id, 'CHI-210828') && 'Editor'
```

Comment:
This line needs work.

mainly we need to be able to template a list of project IDs,
```
- Chameleon
- CHI-210828
- CHI-whatever
```
into some jmespath mapping onto roles. The easy answer is to skip the list idea and just have a "{{ site_operator_project }}" field that can be configured. Except, some of the operators don't have a longrunning project!

See https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/generic-oauth/#jmespath-examples